ui/Screen/Project.svelte

```diff
@@ -60,16 +60,61 @@
   $: revisionStore.set(path.extractProjectSourceRevision($location));
   $: objectPathStore.set(path.extractProjectSourceObjectPath($location));
   $: objectTypeStore.set(path.extractProjectSourceObjectType($location));
+
+  const topbarMenuItems = projectId => [
+    {
+      icon: Icon.Source,
+      title: "Source",
+      href: path.projectSource(projectId)
+    },
+    {
+      icon: Icon.Feed,
+      title: "Feed",
+      href: path.projectFeed(projectId)
+    },
+    {
+      icon: Icon.Fund,
+      title: "Fund",
+      href: path.projectFunds(projectId)
+    },
+    {
+      icon: Icon.Issues,
+      title: "Issues",
+      href: path.projectIssues(projectId)
+    },
+    {
+      icon: Icon.Revisions,
+      title: "Revisions",
+      href: path.projectRevisions(projectId)
+    }
+  ];
 </script>
 
 <SidebarLayout style="margin-top: calc(var(--topbar-height) + 33px)">
   {#await $project then result}
-    <ProjectTopbar
+    <Topbar
       style="position: fixed; top: 0;"
       avatarUrl={result.data.project.metadata.imgUrl}
-      id={result.data.project.id}
-      name={result.data.project.metadata.name} />
-
+      name={result.data.project.metadata.name}
+      href={path.projectOverview(result.data.project.id)}
+      menuItems={topbarMenuItems(result.data.project.id)}>
+      <Flex>
```

Comment:
Re: why wrapping in `Flex` here - I was thinking it'd be useful to vertically center the buttons, but if they look fine without it I'll take it out

Reply:
Vertical centering should already be provided by the `<Topbar>` component.